src/resources/Projects/ProjectInterfaces.ts

```diff
@@ -7,7 +7,7 @@ export enum ProjectSortBy {
     createdDate = 'CREATED_DATE',
     updatedBy = 'UPDATED_BY',
     updatedDate = 'UPDATED_DATE',
-    pointOfContact = 'POINT_OF_CONTACT',
+    pointsOfContact = 'POINT_OF_CONTACT',
```

Comment:
I'm not sure I understand the change, you're fixing a typo?

Reply:
Yes correct! This enum is based off the properties defined lower in the file in `BaseProjectModel` and `ProjectModel`. I fixed a typo in the enum for the `pointsOfContact` property.

<img width="734" alt="image" src="https://github.com/coveo/platform-client/assets/105073504/256d7083-44e2-4e55-bae8-54986c9b2e15">

Reply:
tis a typo, ergo the previous behavior was unwanted and never wanted, thus it's a fix

Reply:
The value accepted by the backend is `POINT_OF_CONTACT` which is not changed by this PR.

If we change a key of the enum it will break implementation using `ProjectSortBy.pointOfContact`, requiring the dev to switch to `ProjectSortBy.pointsOfContact` without changing the actual output of the code

Reply:
Currently, this would not cause any breaking changes because it hasn't been put to use yet (it was added yesterday in this [PR](https://github.com/coveo/platform-client/pull/769)). But as we discussed earlier @GermainBergeron, I can adapt the front-end

Reply:
Derp, Are we sure we want to deviate the camelCase<->UPPER_SNAKE_CASE match/map between the value and the key of the enum?